posts/intro-to-mathport.md

```diff
@@ -0,0 +1,131 @@
+---
+author: 'Scott Morrison'
+category: 'mathport'
+date: 2021-12-11 12:41:00+00:00
+description: ''
+has_math: true
+link: ''
+slug: intro-to-mathport
+tags: ''
+title: Update on mathport (Dec 2021)
+type: text
+---
+
+# An update on mathport
+
+`mathport` is the tool we're planning on using to help us port mathlib to Lean 4.
```

Comment:
Does it make sense to write 2 or 3 sentences for people who have heard of Lean and mathlib, but don't know that we are currently using Lean 3 and that Lean 4 was prereleased this year, etc... Just providing a bit of background info, and sketching what the challenge is: porting 700K lines of code from Lean 3 to Lean 4. Also mathlib keeps growing, we can't freeze it for 4 months.

Reply:
Okay, I've written much more than 2 or 3 sentences. Could you please review again?